src/deepsparse/loggers_v2/root_logger.py

```diff
@@ -0,0 +1,127 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from collections import defaultdict
+from enum import Enum
+from typing import Any, Dict, Optional
+
+from deepsparse.loggers_v2.filters.frequency_filter import FrequencyFilter
+from deepsparse.loggers_v2.filters.pattern import is_match_found
+
+from .utils import import_from_registry
+
+
+class LogType(Enum):
+    SYSTEM = "SYSTEM"
+    PERFORMANCE = "PERFORMANCE"
+    METRIC = "METRIC"
+
+
+class RootLogger(FrequencyFilter):
```

Comment:
what do we get out of having `FrequencyFilter` as a parent class instead of a property that is used as a helper?

Reply:
Depends on how deep the question is. 
Its a class bc you need a state to keep in track of the counter per tag.func. If you mean just a @property in root loggers, yes we can do that. 
If you mean to get rid of FF and combine to RootLogger, we can do that, but better to keep logic isolated